### ISSUE TITLE
Bug potentiel d'affichage des unités dans <Input />

### ISSUE THREAD

**Issue by laem** [betagouv/mon-entreprise#1483]
Voir https://github.com/datagir/nosgestesclimat-site/pull/226

En gros, le code actuel fait une rétroingénierie de formatValue pour en extraire l'unité, plutôt que d'utiliser directement serialize unit. 

La conséquence est visible notamment quand l'unité possède un chiffre, comme "kgCO2e" :) 

Ici https://github.com/betagouv/mon-entreprise/blob/master/mon-entreprise/source/components/conversation/Input.tsx#L25

**Comment by johangirod** [comment]
Effectivement, le comportement actuel est loin d'être idéal... Le problème de la solution `serializeUnit`, c'est qu'elle ne fait pas de pluralisation, alors que `formatValue`, oui.

Je pense néanmoins que c'est la bonne piste, on pourrait imaginer ajouter un argument à `serializeUnit`.

De manière générale, la gestion des unités est à revoir (voir la proposition de #1350). On peut imaginer avoir les fonctions suivantes : 

```js
engine.formatValue(node)
engine.formatUnit(node)
```

A réflechir quand on s'attaquera à la réecriture des unités
